userspace/libsinsp/container_engine/docker/base.cpp

```diff
@@ -33,6 +33,7 @@ docker_base::resolve_impl(sinsp_threadinfo *tinfo, const docker_lookup_request&
 			auto container = sinsp_container_info();
 			container.m_type = request.container_type;
 			container.m_id = request.container_id;
+			container.set_lookup_status(sinsp_container_lookup::state::SUCCESSFUL);
```

Comment:
CRI and docker mostly handled the lookup state correctly, except here it was necessary to add.